src/main/java/io/buildpacks/example/sample/PeerReview.java

```diff
@@ -0,0 +1,21 @@
+// PeerReview.java
```

Comment:
1. Naming Convention: The class name "PeerReview" does not seem to reflect the purpose of the class or the methods inside it. Consider choosing a more descriptive class name that represents the functionality, such as "MathUtils" or similar.